src/js/components/FormField/FormField.js

```diff
@@ -193,7 +185,7 @@ class FormFieldContent extends Component {
             ? { ...border, color: borderColor }
             : undefined
         }
-        margin={abut ? undefined : { ...formField.margin }}
+        margin={abut ? margin : { ...formField.margin }}
```

Comment:
this doesn't sound right to me. i think we should apply margin if not abut. something like `margin={abut ? undefined : margin || { ...formField.margin }}` so that we apply the margin property if user wants, otherwise we keep formField.margin for backwards compatibility.